src/panels/Menu/hooks.ts

```diff
@@ -18,32 +18,41 @@ export function useMenuItems() {
   return { menuItems, filteredMenuItems };
 }
 
+function useShowWindowOnStart(visible: boolean, menuItem: MenuItem) {
+  const { addWindow } = useWindowLayoutProvider();
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    // Show the menu button in the taskbar
+    dispatch(setMenuItemVisible({ id: menuItem.componentID, visible: visible }));
+    if (visible) {
+      // Open the window if it is visible
+      addWindow(menuItem.content, {
+        id: menuItem.componentID,
+        title: menuItem.title,
+        position: menuItem.preferredPosition
+      });
+    }
+  }, [addWindow, dispatch, visible, menuItem]);
+}
+
 export function useStoredLayout() {
   const menuItems = useAppSelector((state) => state.local.taskbarItems);
   const hasMission = useAppSelector((state) => state.missions.isInitialized);
-  const { addWindow } = useWindowLayoutProvider();
+  const hasStartedBefore = useAppSelector((state) => state.profile.hasStartedBefore);
+  const showGettingsStartedTour =
+    hasStartedBefore === undefined ? false : !hasStartedBefore;
+  // Special handling of when the getting started tour should be shown
+  useShowWindowOnStart(showGettingsStartedTour, menuItemsData.gettingStartedTour);
+  // Special handling of when a mission file is loaded
+  useShowWindowOnStart(hasMission, menuItemsData.mission);
```

Comment:
```suggestion
  // Special handling to show getting started tour on startup if first time running
  useShowWindowOnStart(showGettingsStartedTour, menuItemsData.gettingStartedTour);
  // Special handling to show mission panel if the started profile includes a mission file
  useShowWindowOnStart(hasMission, menuItemsData.mission);
```
?  To make it clearer why the special handling is necessary